src/services/Provider.ts

```diff
@@ -194,7 +196,7 @@ export class Provider {
    * @return {Promise<FileInfo[]>} urlDetails
    */
   public async checkFileUrl(
```

Comment:
this method name does not make sense anymore if it checks more than URLs. Based on what method does, `getFileInfo()` might be more fitting name?

Reply:
Done.